docs/proposals/EntGAN.md

```diff
@@ -0,0 +1,33 @@
+# Integrate GAN and Self-taught Learning into Sedna Lifelong Learning to Handle Unknown Tasks
+
+## Motivation
+
+In the process of Sedna lifelong learning, there would be a chance to confront unknown tasks, whose data are always heterogeneous small sample. Generate Adversarial Networks(GAN) is the start-of-art generative model and GAN can generate fake data according to the distribution of the real data. Naturally, we try to utilize GAN to handle small sample problem. Self-taught learning is an approach to improve classfication performance using sparse coding to construct higher-level features with the unlabeled data. Hence, we combine GAN and self-taught learning to help Sedna lifelong learning handle unknown tasks.
+
+### Goals
```

Comment:
See the previous discussion in  https://github.com/kubeedge/sedna/pull/337#discussion_r940166481

The story is not yet completed and it would be much improved with the targeting scenario and dataset added.

Reply:
The current version does not express the limitation of the current lifelong learning. 
1. The current lifelong learning is designed to tackle small sample problems. 
2. Why do we still need a GAN? GAN cannot generate labeled data so far. Why not just set a camera on a car and collect more data?
The author might want to consider adding a related story mentioned in https://github.com/kubeedge/sedna/pull/337/files#r997678113.